public/index.html

```diff
@@ -7,15 +7,16 @@
   <link rel="stylesheet" href="lib/bootstrap/dist/css/bootstrap.min.css"/>
   <link rel="stylesheet" href="lib/angularjs-toaster/toaster.css"/>
 </head>
-<body class="height100">
+<body ng-class="{height100: isLogged, bodyCustom: !isLogged}">
     <!-- navigation bar --> 
-    <nav class="navbar navbar-default navbar-inverse" role="navigation">
+    <nav class="navbar navbar-default navbar-inverse" role="navigation" ng-show="isLogged" ng-controller="LoginCtrl">
```

Comment:
This could be ok for training purposes but in a real application I would prefer to decouple Login controller responsibility from navigation element.